modules/user_accounts/templates/menu_user_accounts.tpl

```diff
@@ -2,7 +2,7 @@
 
 <!-- start the selection table -->
 <div class="row">
-    <div class="col-sm-9 col-lg-8">
+    <div class="col-sm-9 col-lg-12">
```

Comment:
@johnsaigle oh ya whoops, thought this was a general change. Shouldn't we change all of them at the same time?